Competition/src/main/cpp/subsystems/Feeder.cpp

```diff
@@ -72,70 +71,117 @@ void Feeder::assessInputs()
     state.operator_leftBumperPressed = operatorController->GetLeftBumper();
     
 
-    state.lowerBannerTripped = false;//banner_upper.Get();
-    state.upperBannerTripped = false;//banner_upper.Get();
+    state.bannerTripped = !banner.Get();
 
-    if (state.driver_rightBumperPressed || state.operator_leftBumperPressed) {
-        state.feederState = FeederState::FEEDER_INTAKE2;
+    state.currentBanner = state.bannerTripped;
+    
+    if (state.operator_leftBumperPressed) {
```

Comment:
Not code related, but the driver/operator diagrams need to be updated

Reply:
done